data/Jonnie-Dev.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "John Areola",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Software engineer - 👨‍💻 Front end developer || UI Designer",
+  "avatar": "https://github.com/Jonnie-Dev.png",
```

Comment:
`avatar` is not a required property. GH actions auto pick it from your profile. Linking you a complete JSON file https://linkfree.eddiehub.io/docs/full-profile-example